.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: node_js
 node_js:
-  - "4"
+  - "6"
 before_script:
   - make build
+  # even our unit tests require a
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+  - sleep 3 # give xvfb some time to start
```

Comment:
We should be able to remove this if we're using PhantomJS (for now anyway)

Reply:
I will comment out

Reply:
> I will comment out all of this display stuff & leave another FIXME (& note it in the issue)

I'd rather just remove it to keep the code tidy :)

We could add this to the issue if necessary.